docs/src/blockchain-development/storage.md

```diff
@@ -6,6 +6,60 @@ Put in conventional programming terms, contract storage is like saving data to a
 
 Some basic use cases of storage include declaring an owner address for a contract and saving balances in a wallet.
 
+## Storage By-Hand
+
+Outside of the newer experimental `storage` syntax which is being stabalized, you can leverage FuelVM storage operations using the `store` and `get` methods provided in the standard (`std`) library.
+
+With this approach you will have to design the internal key and value design manually.
+
+An example is as follows:
+
+```sway
+contract;
+
+use std::{
+    storage::{get, store}
+};
+
+abi StorageExample {
+    fn store_something(amount: u64);
+    fn get_something() -> u64;
+}
```

Comment:
New code snippets should really be in `examples/` moving forward. Otherwise that'll be even _more_ work to move over.

Reply:
Yes, that makes sense. Will clean this up.